Detectors/EMCAL/workflow/src/CellRecalibratorSpec.cxx

```diff
@@ -57,15 +58,11 @@ void CellRecalibratorSpec::run(framework::ProcessingContext& ctx)
       outputtriggers.push_back(nexttrigger);
```

Comment:
can be done directly w/o intermediate object
```
outputtriggers.emplace_back(trg.getBCData(), currentfirst, trg.getNumberOfObjects()).setTriggerBits(trg.getTriggerBits());
```

Reply:
OK, will change this.

Reply:
Done.